### ISSUE TITLE
Cannot override admin_orm.xml file

### ISSUE THREAD

**Issue by francescomalatesta** [sonata-project/SonataUserBundle#666]
I need to override the admin_orm.xml file in order to remove some menu items that I don't need (I'm handling user management with another service).

The only way I found is to manually remove entries in the vendor admin_orm.xml, but it's quite inelegant.

Any other ideas?

**Comment by cassianotartari** [comment]
One way to configure menu itens as you like is by configuration, for example:

```yaml
sonata_admin:
    dashboard:
        groups:
            group_tag_name:
                items:
                    - your_service.name
```

Or maybe you can override ``SonataUserExtension`` that loads the file, I don't know if it's possible, take a look here: http://symfony.com/doc/current/cookbook/bundles/override.html

**Comment by OskarStark** [comment]
Closed via #869
